src/axes/timeAxis.ts

```diff
@@ -488,6 +489,10 @@ export module Axes {
         this._hideOverlappingAndCutOffLabels(i);
       }
 
+      if (this.annotationsEnabled()) {
+        this._drawAnnotations();
```

Comment:
`Axes.Time` completely overrides `renderImmmediately()` without using the `super` call, which means I need to port the annotation drawing logic there.